main.js

```diff
@@ -0,0 +1,191 @@
+// selet elements in DOM
+
+const form = document.querySelector("#itemForm");
+const itemInput = document.querySelector("#itemInput");
+const itemList = document.querySelector("#itemList");
+const messageDiv = document.querySelector("#message");
+const clearButton = document.querySelector("#clearBtn");
+const filters = document.querySelectorAll(".nav-item");
+
+// create an empty array for tasks(items).
+let todoItems = [];
+
+// showing alert when we add item, or errors.
+const showAlert = function (message, msgClass) {
```

Comment:
Is there a reason to use function keyword and also assign it to a const, instead of just using the normal function syntax like `function nameOfTheFunction(){ ... }` ?

if you want a similar syntax use arrow function:
`const fnName = () => { ... }`